yb-voyager/cmd/importData.go

```diff
@@ -798,6 +801,60 @@ func importData(importFileTasks []*ImportFileTask, errorPolicy importdata.ErrorP
 	fmt.Printf("\nImport data complete.\n")
 }
 
+// For a fresh start but non empty tables in tableList && OnPrimaryKeyConflict is set to IGNORE -> notify user
+func checkPKConflictModeOnFreshStart(importFileTasks, pendingTasks, completedTasks []*ImportFileTask) {
```

Comment:
shall we have a check at the beginning of this function to make sure it  only runs for certain importer-roles - import-data-to-target, import-data-file?